src/test_lib_json/main.cpp

```diff
@@ -3286,11 +3286,11 @@ struct CharReaderAllowDropNullTest : JsonTest::TestCase {
     return [=](const Value& root) { JSONTEST_ASSERT_EQUAL(root, v); };
   }
 
-  ValueCheck objGetAnd(std::string idx, ValueCheck f) {
+  ValueCheck objGetAnd(const std::string idx, const ValueCheck f) {
```

Comment:
I don't want to start adding `const` to locals just to do it. It's extra syntax and doesn't help anything. Code was really quite fine before change.